terraform/cloud-platform-components/resources/fluentd-es/values.yaml

```diff
@@ -0,0 +1,8 @@
+namespace: 
+
+version: 
```

Comment:
I believe we should use `image.repository` and `image.tag` as most of the other charts we use (see [here](https://github.com/helm/charts/blob/master/stable/nginx-ingress/values.yaml#L6-L8)), and provide a default value that is the "supported" version.